main.go

```diff
@@ -39,6 +39,12 @@ func initServer() {
 	serverConfig := config.GetServer()
 	gin.SetMode(gin.DebugMode)
 	router := gin.Default()
+
+	authConfig := config.GetAuth()
+	if err := auth.UpdateJWKS(authConfig.KeysEndpoint); err != nil {
+		log.Fatalf("error getting JWKs: %v", err)
+	}
+	router.Use(auth.JWTAuthMiddleware(authConfig.KeysEndpoint))
 	router.Use(cors.New(cors.Config{
```

Comment:
Does this fail if the user doesn't provide an endpoint?

Reply:
If so, how can local testing be done? This would make the product exclusively tied to OAuth.

Reply:
We probably need to set an env variable through which we can disable OAuth? or @bsekar is there a better way to do integration and acceptance tests?